repaygithub/cactus
# CACTUS-516 :: Preview Component

Ticket: https://repayonline.atlassian.net/browse/CACTUS-516

Some Notes

- I'm not sure how we could indicate to a screen-reader that the image that has focus can be expanded. If anyone knows how we could do that, let me know.
- If you click on the REPAY logo in the preview, you'll see that it looks pretty weird with images that have transparent backgrounds. Should we maybe render a `div` with a white background around the image to help alleviate that?
- Also, this looks pretty bad on mobile. There aren't any mobile designs so we should probably circle back to this when we have those.

### Testing
1. Open the Preview "Basic Usage" story
2. Make sure you can cycle through the images using both your mouse and your keyboard.
3. Make sure you can expand an image both by clicking on it and using your keyboard.
4. Repeat steps 2 & 3 in the "From Children" story.
5. Turn on your screen reader and make sure it works with that. Only the "From Children" story will have descriptions for the images when you focus on them.
